### DISCUSSION TITLE
Urgent: GitHub Account Recovery Assistance Needed

### DISCUSSION THREAD

**Discussion by NikacodeX** [community/community#135256] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

I am facing an extremely distressing situation with my GitHub account. Recently, I was teaching my brother how to navigate GitHub, specifically how to delete a repository. Unfortunately, in a tragic mistake, he ended up deleting my entire GitHub account instead of just the repository.

This account held all of my projects, each one a product of countless hours of effort and dedication. Every piece of code, every milestone reached, was stored on GitHub. I had relied on this platform as the sole repository for all my hard work, believing it to be a safe haven for my projects. I didn't maintain local copies of these projects on my PC, trusting that GitHub would preserve them.

Now, with the account deletion, I find myself in a devastating position where I've lost everything I’ve worked on. The thought of losing all this hard work is overwhelming, and I’m desperately seeking a way to recover my account and restore these invaluable projects. My GitHub username was rubypoddar.

I would deeply appreciate any assistance in helping me recover my account and the projects that were stored there. This is an urgent matter for me, and I am hopeful that GitHub support can help me through this difficult time.

![image](https://github.com/user-attachments/assets/c8e96935-b0b7-413d-8e73-e936375fd163)

**Comment by queenofcorgis** [comment, marked as answer]
Hey there!  👋 

Thanks for posting in the GitHub Community, @NikacodeX ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by ebndev** [comment]
Hi @NikacodeX,

Please see [this response](https://github.com/orgs/community/discussions/135253#discussioncomment-10288831) to a discussion posted by another account about the same issue.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion, and we ask that you no longer cross-post.

Thank you for your understanding!
